taf/testlib/virtual_env.py

```diff
@@ -271,12 +273,17 @@ def ensure_public_access(self, try_reuse=False, networks_client=None, routers_cl
             tenant_id = self.tenant_id
 
         _net_cfg = self.config.network
-        _mgmt_ip_cidr = self.settings.get('mgmt_ip_cidr')
-        assert _mgmt_ip_cidr
-        net_ip = netaddr.IPNetwork(_mgmt_ip_cidr)
+        _external_net_ip_cidr = self.settings.get('external_net_ip_cidr')
+        assert _external_net_ip_cidr
+        net_ip = netaddr.IPNetwork(_external_net_ip_cidr)
+
+        _external_net_pool_start = self.settings.get('external_net_pool_start')
+        assert _external_net_pool_start
+        _external_net_pool_end = self.settings.get('external_net_pool_end')
+        assert _external_net_pool_end
```

Comment:
I did not find those two constants.  Should I put them to virtual_env code? Why is minimum = 100 ?